code_schemes/age.json

```diff
@@ -0,0 +1,1141 @@
+{
```

Comment:
age_category?

Reply:
Thanks, on CSAP we usually add this later once we've checked the distribution. This might be especially important here, since we are explicitly targeting youth.

Reply:
noted thanks.